grails-app/controllers/au/org/ala/ecodata/OrganisationController.groovy

```diff
@@ -114,4 +114,44 @@ class OrganisationController {
         values
     }
 
+    def organisationMetrics(String id) {
+
+        def organisation = Organisation.findByOrganisationId(id)
+
+        boolean approvedOnly = true
+        boolean targetsOnly = false
```

Comment:
Organisations don't have a concept of targets so we should delete targetsOnly and includeTargets